html/partials/compare/changed_config_files.html.haml

```diff
@@ -23,10 +23,14 @@
             - if @diff["changed_config_files"].only_in1
               = only_in_a
               = render_partial "compare/changed_config_files_list", list: @diff["changed_config_files"].only_in1
+            - else
+              No individual changed config files found.
```

Comment:
That's probably a far better approach, I will look into that. 
I chose to add the "individual" to stress that it's a unique pattern etc. that's missing, but that could also be phrased differently of course.